types/mixpanel-browser/index.d.ts

```diff
@@ -175,7 +179,7 @@ export interface Mixpanel {
     track_forms(query: Query, event_name: string, properties?: Dict | (() => void)): void;
     track_links(query: Query, event_name: string, properties?: Dict | (() => void)): void;
     track_with_groups(event_name: string, properties: Dict, groups: Dict, callback?: Callback): void;
-    unregister(property: string): void;
+    unregister(property: string, options?: RegisterOptions): void;
```

Comment:
It seems to have a different use case than the `register` method. How about declaring the `UnregisterOptions` interface and using it?